components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -111,7 +112,9 @@ - (instancetype)initWithTitle:(NSString *)title message:(NSString *)message {
     _header = [[MDCActionSheetHeaderView alloc] initWithFrame:CGRectZero];
     _header.title = [title copy];
     _header.message = [message copy];
-    self.backgroundColor = [UIColor whiteColor];
+    _backgroundColor = UIColor.whiteColor;
+    _header.backgroundColor = UIColor.whiteColor;
+    _tableView.backgroundColor = UIColor.whiteColor;
```

Comment:
Consider either storing UIColor.whiteColor as a local variable or using `_backgroundColor` for the subsequent initializations. This will help communicate that you intend all three of these values to be equal.

Reply:
Done.